src/controllers/teacher.ts

```diff
@@ -89,7 +89,7 @@ export default class TeacherController {
             }
 
         } catch (error) {
-            ctx.body = { message: error.message }
+            console.log(error)
```

Comment:
why remove ctx.body ?

Reply:
i just want to check the error as it is not printing the whole error in postman so i have console logged the error just to debug the error